apps/exchange/lib/exchange/matching_engine.ex

```diff
@@ -201,6 +217,20 @@ defmodule Exchange.MatchingEngine do
     {:reply, {:ok, bid_volume}, order_book}
   end
 
+  def handle_call(:open_orders, _from, order_book) do
+    open_orders =
+      Exchange.OrderBook.open_orders(order_book)
+
+    {:reply, {:ok, open_orders}, order_book}
+  end
+
+  def handle_call(:open_orders_by_trader, trader_id, order_book) do
```

Comment:
It should be
`def handle_call({:open_orders_by_trader, trader_id}, _from, order_book) do`